### DISCUSSION TITLE
Pending License Agreement

### DISCUSSION THREAD

**Discussion by Lurkars** [Lurkars/gloomhavensecretariat#265] [Announcements]
First of all...

## Why don't I have a licence agreement?

In particular, Isaac Childres has stated in the past that all non-commercial community projects are tolerated. Before GHS, there were already a lot of community projects without licence agreements. Also, GHS was released without Frosthaven in mind, so I think the general situation has evolved a bit with all the brand new Frosthaven stuff. Call me naive, but of course I just wanted the best for a non-commercial open source project.

## What has changed now?

Edit: To be fair, I have to admit that nothing has directly changed. The following passage was just a reminder that I was missing a licence agreement and that this situation needs to be resolved!

As of the date of this writing, two days ago, the *Worldhaven* repository [updated its licence terms](https://github.com/any2cards/worldhaven#asset-license-terms) with the following quote in particular

> No other individual, entity, or corporation is permitted to use and/or re-use the assets within this repository for any other purpose.

(see also [related issue](https://github.com/any2cards/worldhaven/issues/20))

This actually ~~tells~~reminds me that I am not allowed to use these assets ~~any more~~. As you can imagine, I rely heavily on the use of artwork assets, while not all assets are directly from *Worldhaven*, it is still impossible for me to indicate what is being used, and in any case, I don't have permission to use assets from other sources, such as the official website.

## Other data licence issue

While artwork assets are an issue regarding the look that can be changed, there is still the open question of the copyright of all texts like on ability cards, monster stats, character stats, scenario titles etc. This also needs to be clarified, as GHS is useless without the ability to use this data.

## Pending status

I have already contacted Cephalofair Games to ask for a settlement, but the automatic reply about delays in support requests (due to Frosthaven Shipment/support) indicates that it will take a while to clarify my situation. In the meantime, I will remain naive and hope to get an agreement soon, or at least be gently told to remove all assets.

## What happens to the GHS without an agreement?

If I get a negative response and am told to remove all assets, I will of course have to do so. My complete source code is written by me and cannot be taken down. So at least that will stay. I will then have to see what can still be used, as most of the basic (non-Frosthaven) assets are from the publicly licensed Creator Pack. I think the assets I need to remove (except for CS I think) are
- all character art
- FC, FH and JOTL monster art
- Special FH artwork/assets like new ability icons, conditions, stickers, backgrounds, sheets, etc.
- big question mark over data: on the old Helper statement it was clear that Cephalofair also claimed the data on ability cards etc, so this would affect most data

### So if all these assets/data are removed...

Around the assets, there are a few options: 
1. keep them removed and try to create a theme that doesn't feel too empty
2. provide a way to load your own assets somehow

For the data there is only one possibility:
1. use the existing way to load external data and use your own data.

So let's hope for the best.

**Comment by Doctawin** [comment]
It's sad to see the potential removal of the assets, but perhaps an agreement can be reached.

**Comment by Lurkars** [comment]
Just a quick update that I still haven't heard anything back. So I guess my mail was ignored/spammed or whatever. Anyway, I haven't heard of any agreements for other community applications either. So this is still open and I hope for the best that it comes with some custom content files like for Gloomhaven that just allow us all to use the assets as is.

**Comment by Lurkars** [comment]
To give a small update: I just rewrote an email and got a quick reply. Unfortunately, the answer was not satisfactory, and I don't know why, if they didn't read my mail correctly or what. But they just referred to the [2017 BBG Thread](https://boardgamegeek.com/thread/1733586/files-creation) that I can use the mentioned Creative Common Licence from there. Of course, this licence does not include any assets other than the base of Gloomhaven (so besides Frosthaven, I also lack an agreement for Jaws of the Lion and Forgotten Circles), nor does this licence affect character artwork at all. So I wrote back to them and specifically asked for this stuff. I haven't heard back yet. So this process is still ongoing.
